### ISSUE TITLE
jq: error: col/1 is not defined

### ISSUE THREAD

**Issue by alexangas** [luckman212/1password-csv-fix#2]
When running `./1password-csv-fix.sh --fix-multi`

I get the error:

```
jq: error: col/1 is not defined at <top-level>, line 2:
        [ .id, (.title|col(25)), .urls[0].href ] | @tsv                
jq: 1 compile error
```

I'm not familiar with jq so not sure what this means. If it helps, the version installed on my system is jq-1.7.1.

**Comment by luckman212** [comment]
Ah. Sorry about that! Thought I caught all the dependencies, but I forgot about a few custom jq functions that I'd defined.

I'll upload a corrected script shortly!

**Comment by luckman212** [comment]
Please try [v1.0.1](https://github.com/luckman212/1password-csv-fix/releases/tag/1.0.1) and let me know how it goes.

**Comment by alexangas** [comment]
Fixed! Thanks again!
